preppal-fe/src/components/nav-bar/nav-bar.tsx

```diff
@@ -20,7 +38,7 @@ const NavBar = () => {
       <Nav className="links">
         <Nav.Link className="nav-item" href="collections">Collections</Nav.Link>
         <Nav.Link className="nav-item" href="new-recipe">New Recipe</Nav.Link>
-        {loggedIn
+        {{loggedIn}
           ? (<Nav.Link className="nav-item" href="profile">Profile</Nav.Link>)
```

Comment:
We probably also want to include new recipe and collections as well since they both require a user

Reply:
These have been added. They didn't want to be in one loggedIn check, so I made three.